docs/dummy/README.md

```diff
@@ -0,0 +1,2 @@
+// This crate is just used to provide the needed dependencies
+// for the mdbook documentation tests
```

Comment:
```suggestion
This crate is only used to provide the needed dependencies for the mdbook documentation tests.
```

This is a normal markdown file, the comment syntax looks strange when it is rendered.